src/base-cookie.js

```diff
@@ -47,6 +67,19 @@ const set = (name, value, opts) => {
   document.cookie = str;
 };
 
+const getLastEventTime = (cookie) => {
+  const strValue = cookie.split('.')[Constants.LAST_EVENT_TIME_INDEX];
+  return parseInt(strValue, 32) || 0;
+};
+
+const sortByEventTime = (cookies) => {
+  return cookies.sort((c1, c2) => {
+    const t1 = getLastEventTime(c1);
+    const t2 = getLastEventTime(c2);
+    return t1 > t2 ? c1 : c2;
```

Comment:
haven't tested this yet, but does this work? I thought [sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description) took a function that returned -1, 0, or 1 depending on the sort order